platformics/api/core/helpers.py

```diff
@@ -69,8 +69,6 @@ def convert_where_clauses_to_sql(
         raise PlatformicsException("Max filter depth exceeded")
     depth += 1
 
-    if not whereClause and not order_by and not group_by:
-        return query, [], []
```

Comment:
We can't return early anymore since we need to add the clauses to filter out soft-deleted data.